### PR TITLE
Update pkg description for Microsoft.Azure.ServiceBus to include pointer to new package

### PR DESCRIPTION
**Pull request by ramya-rao-a**
This PR updates the package description for `Microsoft.Azure.ServiceBus` package to include pointer to the new package and the migration guide

As discussed offline, this is in an attempt to guide users to the new package where there is active development while the above packages continue to get critical bug fixes

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Azure ServiceBus SDK</AssemblyTitle>
-    <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <Version>5.2.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.ServiceBus is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/sb for more details.</Description>
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]: Feb 2020?

Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]:
```suggestion
    <Description>Please note, a newer package Azure.Messaging.ServiceBus is available as of November 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/sb for more details.</Description>
```

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj (hunk 1 above)
Comment (ramya-rao-a) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj]: copy paste error from the same PR for Event Hubs!!

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Azure ServiceBus SDK</AssemblyTitle>
-    <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <Version>5.2.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.ServiceBus is available as of November 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/sb for more details.</Description>
```
Comment (jsquire) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]:
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.ServiceBus, was released in November 2020. While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Please see the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

File: sdk/servicebus/Microsoft.Azure.ServiceBus/README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Azure Service Bus client library for .NET
 
+> Please note, a newer package [Azure.Messaging.ServiceBus](https://www.nuget.org/packages/Azure.Messaging.ServiceBus) for [Azure Service Bus](https://azure.microsoft.com/services/service-bus/) is available as of December 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the [migration guide](https://aka.ms/azsdk/net/migrate/sb) for more details.
```
Comment (jsquire) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/README.md:3]:
```suggestion
> Please be aware that a newer package, [Azure.Messaging.ServiceBus](https://www.nuget.org/packages/Azure.Messaging.ServiceBus) is available as of November 2020. While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development.  Please see the [migration guide](https://aka.ms/azsdk/net/migrate/sb) for more details.
```

Comment (jsquire) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/README.md:3]: Was it November or December?   The README says one thing, the package description another.  😄

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj (hunk 2 above)
Comment (jsquire) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]: In my opinion, the `for new development` is a significant distinction.  While we'd like to see migration in existing applications, I don't believe there is forcing function for it nor that there is enough of a value proposition to justify the efforts for many applications with an existing investment in the legacy packages.

**Review by DorothySun216 — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj (hunk 2 above)
Comment (ramya-rao-a) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]: I believe @elraikhm went through an approval process for the current message. @elraikhm, would you be ok with @jsquire's recommendation above?

Comment (elraikhm) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj:4]: @jsquire, I understand your reasoning, but we would like to recommend upgrading to the new package when possible for existing apps as well.  The app can live for years and customers will need to upgrade eventually to stay current and to receive the latest features and fixes.

**Pull request merged**